### PR TITLE
mergify: close automated prs if they are obsoleted

### PR DESCRIPTION
**Pull request by v1v**
## What is the problem this PR solves?

PRs that were created by the automation then close them if they are obsoleted.

They are obsoleted if they have a conflict.

## Related issues

- Relates https://github.com/elastic/beats/pull/29455

### CODE REVIEW

**Review by jlind23 — APPROVED** [review]

**Review by jlind23 — CHANGES_REQUESTED** [review]
dash missing on author value.

**Review by jlind23 — APPROVED** [review]
After reading mergify doc, looks good to me.

File: .mergify.yml [hunk 1]
```diff
@@ -6,7 +6,10 @@ queue_rules:
 pull_request_rules:
   - name: ask to resolve conflict
     conditions:
+      - -merged
```
Comment (v1v) [inline .mergify.yml:9]: `-` is the negative condition in Mergify language :)

**Comment by jlind23** [comment]
Will old PR be automatically closed or should I clean them up?

**Comment by v1v** [comment]
> Will old PR be automatically closed or should I clean them up?

I think they won't be closed since the `conflict` event already happened. Though, I'll do close them since I wanna verify this changes once they are merged

**Pull request merged**